net-mail/getmail/getmail-6.03.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_REQ_USE="ssl?"
+
+DISTUTILS_USE_SETUPTOOLS=no
+inherit distutils-r1
+
+DESCRIPTION="A mail retriever with reliable Maildir and mbox delivery"
+HOMEPAGE="http://www.getmail6.org/"
+SRC_URI="https://github.com/getmail6/getmail6/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~ppc-macos ~x86-macos ~x64-solaris"
+IUSE="ssl"
+
+S="${WORKDIR}/getmail6-${PV}"
+
+python_prepare_all() {
+	sed -i -e "s,'getmail-%s' % __version__,'${PF}'," \
```

Comment:
Please add a comment above this `sed` call to describe what it does. Apparently just renames doc directory / doc file?

Reply:
I have a question about this. The sed call (on line 24) deletes the COPYING file. Is this per a gentoo policy to avoid a million copies from being installed? If so, where can I find it? The code refers to the COPYING files in many places, so I wonder if leaving this deletion in isn't actually going to cause more confusion over just leaving it in.

I have "recycled" the ebuild file that was used for the getmail 5.14 package when it was still included in the distribution and used that as a starting point.